source/StepBro.ExecutionHelper/ExecutionHelperMessages.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StepBro.ExecutionHelper.Messages
+{
+    internal class ExecutionHelperMessages
+    {
+    }
+
+    public enum ShortCommand
+    {
+        None,
+        Close,
+        Acknowledge
```

Comment:
Acknowledge of what?

Reply:
Yup

Reply:
Well, technically acknowledge that everything went well